src/main/java/org/broadinstitute/hellbender/engine/spark/datasources/ReadsSparkSource.java

```diff
@@ -40,39 +40,83 @@
 public final class ReadsSparkSource implements Serializable {
     private static final long serialVersionUID = 1L;
     private static final String HADOOP_PART_PREFIX = "part-";
-    public static final int DEFAULT_SPLIT_SIZE = 10485760;
+    public static final int DEFAULT_SPLIT_SIZE = 10 * 1024 * 1024; // 10MB
```

Comment:
If we're using 128 MB for the input split size, where does this 10 MB come in (apart from the "recommended parallelism" computation)? If it's only really used on output it should probably not live in `ReadsSparkSource`